python_as4.py

```diff
@@ -0,0 +1,138 @@
+class BankAccount:                  #Parent class for bank account
```

Comment:
Good to see a class which is not Calculator. But add more logics to the code. add try exception or if condition check